packages/app/src/systems/Network/hooks/useNetworkForm.ts

```diff
@@ -15,9 +15,10 @@ function isValidUrl(url?: string) {
   if (!url) return false;
   // Note: new URL('https://graphql') returns `true`
   const pattern = new RegExp(
-    '^(https?:\\/\\/)?' +
+    '^(https?:\\/\\/)' +
```

Comment:
> Since we removed the ? we require the urls to be prefixed by some sort of http: or https: prefix?

Yes, I think we should keep the protocol mandatory for the provider url

> Doesn't this pattern allow for urls like "https:\\"? is this the desired behavior?

`^(https?:\\/\\/)`=> Must start with `http` or `https`, `s?` means `s` is optional. `\\` is used to escape special characters when using `new RegExp()`. Regex will evaluate `https:\\` as false.
ref: https://stackoverflow.com/a/17863171